src/RetrySettings.php

```diff
@@ -102,13 +102,13 @@
  * method:
  * ```
  * $customRetrySettings = new RetrySettings([
- *     'initial_retry_delay_millis' => 100,
- *     'retry_delay_multiplier' => 1.3,
- *     'max_retry_delay_millis' => 60000,
- *     'initial_rpc_timeout_millis' => 20000,
- *     'rpc_timeout_multiplier' => 1.0,
- *     'max_rpc_timeout_millis' => 20000,
- *     'total_timeout_millis' => 600000,
+ *     'initialRetryDelayMillis' => 100,
+ *     'retryDelayMultiplier' => 1.3,
+ *     'maxRetryDelayMillis' => 60000,
+ *     'initialRpcTimeoutMillis' => 20000,
+ *     'rpcTimeoutMultiplier' => 1.0,
+ *     'maxRpcTimeoutMillis' => 20000,
+ *     'totalTimeoutMillis' => 600000,
```

Comment:
This example should include `retryableCodes`, since this is a required parameter and no example exists.